pymtl3/passes/sverilog/import_/ImportConfigs.py

```diff
@@ -0,0 +1,452 @@
+      # These options will be passed to verilator to generate the C simulator.
+      # By default, verilator is called with `--cc`.
+
+      # --top-module
+      # Expects the name of the top component;
+      # "" to use name of the current component to be imported
+      "top_module" : "",
+
+      # Expects path of the file to be verilated
+      # "" to use "<top_module>.sv"
+      "vl_src" : "",
+
+      # --Mdir
+      # Expects the path of Makefile output directory;
+      # "" to use `obj_dir_<top_module>`
+      "vl_mk_dir" : "",
+
+      # -f
+      # Expects the path to the flist file; "" to disable this option
+      "vl_flist" : "",
+
+      # -I ( alias of -y and +incdir+ )
+      # Expects a list of include paths; [] to disable this option
+      "vl_include" : [],
+
+      # --assert
+      # Expects a boolean value
+      "enable_assert" : True,
+
+      # Verilator optimization options
+
+      # -O0/3
+      # Expects a non-negative integer
+      # Currently only support 0 (disable opt) and 3 (highest effort opt)
+      "vl_opt_level" : 3,
+
+      # --unroll-count
+      # Expects a non-negative integer
+      # 0 to disable this option
+      "vl_unroll_count" : 1000000,
+
+      # --unroll-stmts
+      # Expects a non-negative integer
+      # 0 to disable this option
+      "vl_unroll_stmts" : 1000000,
+
+      # Verilator warning-related options
+
+      # False to disable the warnings, True to enable
+      "vl_W_lint" : True,
+      "vl_W_style" : True,
+      "vl_W_fatal" : True,
+
+      # Un-warn all warnings in the given list; [] to disable this option
+      # The given list should only include strings that appear in `Warnings`
+      "vl_Wno_list" : [],
+
+      # Verilator misc options
+
+      # --trace
+      # Expects a boolean value
+      "vl_trace" : False,
+
+      # Passed to verilator tracing function
+      "vl_trace_timescale" : "10ps",
+
+      # C-compilation options
+      # These options will be passed to the C compiler to create a shared lib.
+
+      # Additional flags to be passed to the C compiler.
+      # By default, CC is called with `-O0 -fPIC -shared`.
+      # "" to disable this option
+      "c_flags" : "",
+
+      # Additional include search path of the C compiler.
+      # [] to disable this option
+      "c_include_path" : [],
+
+      # Additional C source files passed to the C compiler.
+      # [] to compile verilator generated files only.
+      "c_srcs" : [],
+
+      # `LDLIBS` will be listed after the primary target file whereas
+      # `LDFLAGS` will be listed before.
+
+      # We enforce the GNU makefile implicit rule that `LDFLAGS` should only
+      # include non-library linker flags such as `-L`.
+      "ld_flags" : "",
+
+      # We enforce the GNU makefile implicit rule that `LDLIBS` should only
+      # include library linker flags/names such as `-lfoo`.
+      "ld_libs" : "",
+    }
+
+    s.PassName = 'sverilog::ImportPass'
```

Comment:
Will Python style instead of c++ be better here?

Reply:
I agree